apis/python/src/tiledb/vector_search/module.py

```diff
@@ -36,15 +36,15 @@ def load_as_matrix(
     a = tiledb.ArraySchema.load(path, ctx=tiledb.Ctx(config))
     dtype = a.attr(0).dtype
     if dtype == np.float32:
-        m = tdbColMajorMatrix_f32(ctx, path, 0, 0, 0, size, 0, timestamp)
+        m = tdbColMajorMatrix_f32(ctx, path, 0, 0, 0, size, 0, timestamp, True)
```

Comment:
Let avoid changing this if not needed

Reply:
So I did this because I didn't want to change the behavior of the existing Python implementation at all, and this is the flag which keeps the old implementation. It would be cleaner looking to set the default value for this argument (`bool read_full_matrix` in `src/include/detail/linalg/tdb_matrix.h`) to `True`, but I didn't set a default value because I'd like to migrate to not needing `bool read_full_matrix` at all (and instead having callers pass in the size to read correctly), and in the meantime I want it to be very explicit whether we are doing `read_full_matrix` or not.

Reply:
Changed in the update so will resolve this and re-request your review!